certified-connectors/GreenSign/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "GreenSign",

```

Comment:
identityProvider should be any one of the below. 

 "identityProvider": {
          "type": "string",
          "enum": [
            "oauth2",
            "oauth2generic",
            "aad",
            "aadcertificate",
            "facebook"
          ]

Please refer the schema https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

Reply:
I have made the requested changes. Please re-review.